### DISCUSSION TITLE
sensor.mail_image_url and Nabu Casa

### DISCUSSION THREAD

**Discussion by bmitchell76** [moralmunky/Home-Assistant-Mail-And-Packages#970] [Q&A] [answered]
Hello, I'm using Nabu Casa and am trying to set some alerts but sensor.mail_image_url shows unknown.  I see in the documentation that either `External_URL` or `Internal_URL` is defined, but since I'm using Nabu Casa, I'm unsure how to do so... Maybe I need a webkook?

**Comment by firstof9** [comment]
External_URL or Internal_URL will be pulled from your HA automatically.
Your external_url should be whatever your nabu casa URL is.

**Reply by bmitchell76** [reply]
Thanks but I'm missing something, mail and packages is returning sensor.mail_image_url as unknown.  If I try and access the sensor.mail_image_system_path appended to the nabu case URL I have to sign in making the URL unusable for an iOS push notification.

**Reply by firstof9** [reply]
There's a camera entity use that.

**Reply by bmitchell76** [reply]
Thanks for your help, I finally got back to this.  I have the iOS notification working with the camera entity but I only get a thumbnail of the first piece of mail.  I added the content-type: video

entity_id: camera.mail_usps_camera
content-type: video

**Reply by firstof9** [reply]
I'm not sure what you could do with iOS I use Android.

**Reply by bmitchell76** [reply, marked as answer]
got it...  thanks for getting me in the right direction with the camera entity. 

action: notify.mobile_app_iphone
data:
  message: Mail Arriving Today
  title: USPS Informed Delivery
  data:
    attachment:
      content-type: jpg
    push:
      badge: 0
      category: camera
    entity_id: camera.mail_usps_camera
